tests/fast-integration/Makefile

```diff
@@ -57,5 +57,24 @@ ci-application-connectivity-2-compass:
 
 .PHONY: ci-test-eventing
 ci-test-eventing:
-	npm ci
-	npm run test-eventing
+	npm install
+	npm run eventing-upgrade-test-prep
+	npm run eventing-upgrade-test-tests
+	npm run eventing-upgrade-test-cleanup
+
+.PHONY: ci-test-eventing-pre-upgrade
+ci-test-eventing-pre-upgrade:
+	npm install
+	npm run eventing-upgrade-test-prep
+	npm run eventing-upgrade-test-tests
+
+.PHONY: ci-test-eventing-pre-upgrade
+ci-test-eventing-post-upgrade:
+	npm install
+	npm run eventing-upgrade-test-tests
+	npm run eventing-upgrade-test-cleanup
+
+.PHONY: ci-test-eventing-tests
+ci-test-eventing-post-tests:
```

Comment:
use the same name for the Phony target name.

I suggest `ci-test-eventing-run`